### DISCUSSION TITLE
Awesome

### DISCUSSION THREAD

**Discussion by artydev** [jeasx/jeasx#7] [General]
Hy,
I hope  **jeasx**  be renowed as it derseves.
You made a great work.
Thank you
Regards

**Comment by jablonski** [comment]
Hi,

thank you for your kind words, much appreciated.

Just a curious question: What is it what you like about Jeasx?

Cheers, Maik

**Comment by artydev** [comment]
Mainly its simplicity.
I am envolved in a project which use NextJS, and there is absolutely no doubt, If it was possible,
I would shwicth to Jeasx

Regards

**Reply by jablonski** [reply]
Thanks for the update, I feel your pain as I'm currently involved in some Next-JS-projects too.
